### ISSUE TITLE
Critical Error

### ISSUE THREAD

**Issue by KrocM** [NitinPrakash/wc-thanks-redirect#2]
Hello NitinPrakash

I had some trouble with this plugin today, it blocked me from accessing my DB and dashboard with a critical error. The server host gave me the following info below: 

[22-Dec-2022 14:03:57 UTC] PHP Fatal error:  requireonce(): Failed opening required '/home3/kenmerc1/publichtml/wp-content/plugins/wc-thanks-redirect/freemius/includes/managers/class-fs-clone-manager.php' (includepath='.:/opt/cpanel/ea-php74/root/usr/share/pear') in /home3/kenmerc1/publichtml/wp-content/plugins/wc-thanks-redirect/freemius/require.php on line 24

Is there a work around to continue using this please? For the moment I have disabled it. 
Kind regards,

KrocM

**Comment by NitinPrakash** [comment]
Please take backup.

Deactivate wc-thanks-redirect, delete the wc thanks redirect plugin. Then
re-install the plugin from WordPress.Org repository.

Please let me know.

**Comment by KrocM** [comment]
Thank you for your quick response! I noticed there were changed files in my website and an advisory about your plugin on the Wordfence dashboard, I repaired the files and then was unable to access anything at all, just seeing 'critical error'. The techs at Bluehost were able to get me back online again and, trying everything you advised seems to be working now. If I notice anything else I will keep you in the loop. Thanks for your work on this plugin, very much appreciated!
